ext/web/12_location.js

```diff
@@ -180,19 +180,20 @@ class Location {
         enumerable: true,
       },
       [SymbolFor("Deno.privateCustomInspect")]: {
-        value: function (inspect) {
-          const object = {
-            hash: this.hash,
-            host: this.host,
-            hostname: this.hostname,
-            href: this.href,
-            origin: this.origin,
-            pathname: this.pathname,
-            port: this.port,
-            protocol: this.protocol,
-            search: this.search,
-          };
-          return `${this.constructor.name} ${inspect(object)}`;
+        value: function (inspect, inspectOptions) {
+          return `${this.constructor.name} ${
+            inspect({
+              hash: this.hash,
+              host: this.host,
+              hostname: this.hostname,
+              href: this.href,
+              origin: this.origin,
+              pathname: this.pathname,
+              port: this.port,
+              protocol: this.protocol,
+              search: this.search,
+            }, inspectOptions)
+          }`;
```

Comment:
Since `location` is a special object, I didn't dare to use `createFilteredInspectProxy`.